phys/module_cu_ntiedtke.F

```diff
@@ -585,6 +590,18 @@ subroutine tiecnvn(pu,pv,pt,pqv,pqc,pqi,pqvf,ptf,poz,pzz,pomg, &
       integer i,j,k,lq,km,km1
       real dt,ztpp1
       real zew,zqs,zcor
+      real scale_fac(lq), dxref
+!
+!  set scale-dependency factor when dx is < 15 km
+!
+      dxref = 15000.
+      do j=1,lq
+      if (dx(j).lt.dxref .and. ntiedtke_dx_opt.eq.1) then
+          scale_fac(j) = (1.+log(dxref/dx(j)))**3
+      else
+          scale_fac(j) = 1.+1.33e-5*dx(j)
```

Comment:
I think this function is discontinuous at 15 km.

Reply:
also this function increases with dx > 15 km and decreases when < 15 km

Reply:
@dudhia Here are two plots of the scale factors: The first has two lines one for each of the scale_fac equations (red is the new one).
<img width="634" alt="scale1" src="https://user-images.githubusercontent.com/12705680/212224032-4a71232b-6883-4f24-99b1-cfe85b4300ef.png">

The second one is an enlarged one for the lower values of scale_fac, so that you can see the two functions when it crosses 15 km. I would argue that the continuity isn't bad. The original function does increases with DX, but it changes only very slightly with DX.

<img width="635" alt="scale2" src="https://user-images.githubusercontent.com/12705680/212224436-c8832660-d5b9-4b9e-90b3-626f035cfd69.png">